projects/benqi-lending/README.md

```diff
@@ -0,0 +1,39 @@
+# benqi-lending
+
+Integration with [Benqi Lending Protocol](https://app.benqi.fi/markets)
+
+## Supported Networks
+
+- AVALANCHE
+
+## Common Tasks
+
+1. Basic Operations
+
+- "Deposit 100 AVAX as collateral in @benqi-lending on Avalanche network"
+- "Execute borrow operation for 50 USDC tokens in @benqi-lending"
+- "Enter AVAX market in @benqi-lending"
+- "Repay 100 USDC tokens borrowed in @benqi-lending"
+- "Withdraw 1000 AVAX collateral from @benqi-lending"
+- "Borrow 500 AVAX tokens with 1000 AVAX collateral in @benqi-lending"
+- "Exit AVAX market in @benqi-lending"
+- "What is my current liquidity in @benqi-lending?"
+- "How much can I borrow in @benqi-lending for AVAX market?"
+- "Get my borrow limit for USDC market in @benqi-lending"
+
+## Available Functions
+
+- `borrow`: Borrow a specified amount of an asset in @benqi-lending
+- `depositCollateral`: Deposit a specified amount of an asset as collateral in @benqi-lending
+- `enterMarkets`: Enter a list of markets for specified assets in @benqi-lending
+- `exitMarket`: Exit a market for a specified asset in @benqi-lending
+- `getAccountLiquidity`: Get account liquidity in USD
+- `getMarketBorrowLimit`: Get specified market borrow limit
+- `repayBorrow`: Repay a borrowed amount of an asset in @benqi-lending
+- `withdrawCollateral`: Withdraw a specified amount of collateral from @benqi-lending
```

Comment:
Same story with arguments might help here for transparency 😉

Reply:
See https://github.com/RealWagmi/anon-integration-guide/pull/16#discussion_r1935260535